clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -108,6 +113,16 @@ def reload_client(self) -> None:
             else None
         )
 
+    def set_label_selectors(self, label_selectors: Collection[str], add_to_existing: bool = False) -> None:
+        """Set label selectors for node listing purposes
+
+        :param Collection[str] label_selectors: list of selectors (joined with logic and)
+        :param bool add_to_existing: if set add to existing selectors rather than replacing
+        """
+        self._label_selectors = sorted(
```

Comment:
nit: does this need to be sorted? seems like we could `s/sorted/list` and save some cpu cycles :)

Reply:
It doesn't really need to be, but since it's a bit which can be useful to be human readable, for example in k8s audit logs, it'll be easier if the labels are not shuffled every time due to `set`. And it's going to be used very sporadically, with surely less than 4 strings in the set, so negligible performance impact.